include/deal.II/base/std_cxx11/type_traits.h

```diff
@@ -77,6 +79,9 @@ namespace std_cxx11
                               boost::has_trivial_constructor<T>::value &&
                               boost::has_trivial_destructor<T>::value;
   };
+
+  using boost::true_type;
+  using boost::false_type;
```

Comment:
No, but I used a reference implementation of that file, hoping that this will work.
We already have the `#include <boost/type_traits.hpp>` which is including the file `boost/type_traits/integral_constant.hpp`, in which the typedef is done.